synapse/tools/dumprows.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+synapse - dumprows.py
+Created on 7/24/17.
+
+Dump all of the rows from a cortex storage instance.
+"""
+# Stdlib
+import os
+import sys
+import json
+import time
+import logging
+import argparse
+
+# Third Party Code
+# Custom Code
+import synapse
+import synapse.axon as s_axon
+import synapse.common as s_common
+import synapse.compat as s_compat
+import synapse.cortex as s_cortex
+
+import synapse.lib.tufo as s_tufo
+import synapse.lib.output as s_output
+
+logger = logging.getLogger(__name__)
+
+ADDROWS = 'core:save:add:rows'
```

Comment:
since "ADDROWS" isn't really any easier to find and replace than "core:save:add:rows", i think i'd rather we stick to using the immediates directly ( like the splices/events in the cortex layer ) for the slight clarity/speed benefit.